pyvista/plotting/renderer.py

```diff
@@ -695,6 +695,32 @@ def has_charts(self):  # numpydoc ignore=RT01
         """Return whether this renderer has charts."""
         return self._charts is not None and len(self._charts) > 0
 
+    @property
+    def charts(self):  # numpydoc ignore=RT01
+        """Return a list of all charts in this renderer.
+
+        Examples
+        --------
+        .. pyvista-plot::
+           :force_static:
+
+           >>> import matplotlib.pyplot as plt
```

Comment:
I would simplify this example, to focus on the `get_charts` method.
E.g. a simple `Chart2D` with a line plot, adding the chart to the plotter, and showing that `chart is pl.renderer.get_charts()[0]` evaluates to `True`?

Reply:
Done, thanks